scripts/migration_prod_staging.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
+#
+# Migrate from prod stack to staging stack.
+#
+# TZ=US/Pacific
+# H 21 * * 3-5
+#
+
+
+set +x
+
+# dev or prod
+STACK=${1}
+
+# Folder containing source code
+PATH=${2}
+
+cd $PATH
+
+mvn clean install
+
```

Comment:
We should run the migration twice. First time with 'org.sagebionetworks.remain.read.only.mode=true' and second time with 'org.sagebionetworks.remain.read.only.mode=false'